microprofile-opentracing/src/test/java/org/jboss/eap/qe/microprofile/opentracing/v10/MicroProfileOpenTracing10Test.java

```diff
@@ -0,0 +1,173 @@
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.junit.InSequence;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.eap.qe.microprofile.opentracing.MicroProfileOpenTracingServerHelper;
+import org.jboss.eap.qe.ts.common.docker.Docker;
+import org.jboss.eap.qe.ts.common.docker.DockerContainers;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import io.opentracing.Tracer;
+import io.restassured.RestAssured;
+
+@RunWith(Arquillian.class)
+public class MicroProfileOpenTracing10Test {
+
+    private static final String APPLICATION_NAME = MicroProfileOpenTracing10Test.class.getSimpleName();
+
+    @ClassRule
+    public static Docker jaegerContainer = DockerContainers.jaeger();
+
+    @ArquillianResource
+    private URL baseApplicationUrl;
+
+    @BeforeClass
+    public static void reloadServer() throws Exception {
+        MicroProfileOpenTracingServerHelper.reload();
+    }
+
+    @Deployment
+    public static Archive<?> deployment() {
+        String mpConfig = "sampler.param=1\n" + // sample every trace
+                "sampler.type=const\n"; // use the same decision strategy for all traces
+
+        return ShrinkWrap.create(WebArchive.class, APPLICATION_NAME + ".war")
+                .addPackage(RestApplication.class.getPackage())
+                .addPackage(Tracer.class.getPackage())
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(new StringAsset(mpConfig), "microprofile-config.properties");
+    }
+
+    /**
+     * @tpTestDetails Send a request to deployed application on /rest endpoint.
+     * @tpPassCrit Deployed application is available and returns expected response.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @InSequence(1)
+    @RunAsClient
+    public void applicationRequest() {
+        String response = RestAssured.when().get(baseApplicationUrl + "rest").asString();
+        assertThat(response, equalTo("Hello from traced service"));
+    }
+
+    /**
+     * @tpTestDetails Jaeger server should have recorded traces initiated by
+     *                the request from applicationRequest() test.
+     * @tpPassCrit Traces are recorded on Jaeger server and associated with the deployed application.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @InSequence(2)
+    @RunAsClient
+    public void trace() {
+        // the tracer inside the application doesn't send traces to the Jaeger server immediately,
```

Comment:
Can all `@Test` have `test...` in name, please?

Reply:
fixed, thanks